File: crates/deslop/tests/fixtures/fused-golden-php/shape_beta.php
<?php

class BillingDescriptor
{
    public string $region = 'beta-billing';
    public int $attempts = 9;
    public string $origin = 'https://beta.example.com/billing';

    public function assemble(string $account): string
    {
        return $this->origin . '/' . $account . '/' . $this->region . '?attempts=beta';
    }
}

File: crates/deslop/tests/fixtures/fused-golden-csharp/shape_beta.cs
namespace Golden.Shapes;

public sealed class BillingDescriptor
{
    public string Region { get; } = "beta-billing";
    public int Attempts { get; } = 9;
    public string Origin { get; } = "https://beta.example.com/billing";

    public string Assemble(string account)
    {
        return Origin + "/" + account + "/" + Region + "?attempts=" + Attempts;
    }
}
